src/app/src/components/Dashboard.jsx

```diff
@@ -81,107 +99,60 @@ function Dashboard({ userWithAccessHasSignedIn, fetchingSessionSignIn }) {
         </div>
     );
 
-    const makeClickableDashboardLinkFn = screenTitle => () => (
-        <span>
-            <Link to={dashboardRoute} href={dashboardRoute}>
-                Dashboard
-            </Link>
-            {` / ${screenTitle}`}
-        </span>
-    );
-
     return (
         <AppOverflow>
             <AppGrid
                 style={dashboardStyles.appGridStyles}
                 title={
-                    <Switch>
-                        <Route
-                            exact
-                            path={dashboardListsRoute}
-                            render={makeClickableDashboardLinkFn(
-                                'Contributor Lists',
-                            )}
-                        />
-                        <Route
-                            exact
-                            path={dashboardClaimsDetailsRoute}
-                            render={() => (
-                                <FeatureFlag
-                                    flag={CLAIM_A_FACILITY}
-                                    alternative={DASHBOARD_TITLE}
-                                >
-                                    {makeClickableDashboardLinkFn(
-                                        'Facility Claim Details',
-                                    )()}
-                                </FeatureFlag>
-                            )}
-                        />
-                        <Route
-                            exact
-                            path={dashboardClaimsRoute}
-                            render={() => (
-                                <FeatureFlag
```

Comment:
In general I like how Breadcrumbs dried things up but share your concerns about configurability. We don't actually need these specific `FeatureFlag`s any more, but we would be blocked from easily adding one in the future.